go.mod

```diff
@@ -57,3 +56,5 @@ require (
 	gopkg.in/sohlich/elogrus.v3 v3.0.0-20180410122755-1fa29e2f2009
 	gopkg.in/toast.v1 v1.0.0-20180812000517-0a84660828b2 // indirect
 )
+
+replace github.com/deepmap/oapi-codegen v1.3.5 => github.com/winder/oapi-codegen v1.3.5-algorand
```

Comment:
it's very nice that you forked it.. but if it's still needed, it should go into algorand/oapi-codegen rather then winder/oapi-codegen

Reply:
yes, done.